de.tubs.variantsync.core/src/de/tubs/variantsync/core/managers/ConfigurationProjectManager.java

```diff
@@ -115,7 +119,8 @@ public ConfigurationProject getConfigurationProject(IFeatureProject project) {
 	}
 
 	private void findVariants(ConfigurationProject configurationProject) {
-		for (IFile file : configurationProject.getFeatureProject().getAllConfigurations()) {
+		for (Path path : configurationProject.getFeatureProject().getAllConfigurations()) {
+			IFile file = (IFile) EclipseFileSystem.getResource(path);
```

Comment:
Is this type cast always possible?

Reply:
In my opinion this type cast should be always possible because the `getAllConfigurations()` method should check wether the file is supported, i.e. if it is a valid configuration file. And config files can be treated as `IFile`.
Furthermore this type cast is used even in FeatureIDE, e.g. in `plugins/de.ovgu.featureide.core.ahead/src/de/ovgu/featureide/ahead/AheadComposer.java`